extension/js/common/composer.ts

```diff
@@ -1919,7 +1919,14 @@ export class Composer {
       this.S.cached('icon_show_prev_msg').show().addClass('progress');
       Xss.sanitizeAppend(this.S.cached('icon_show_prev_msg'), '<div id="loader">0%</div>');
       this.resizeComposeBox();
-      this.messageToReplyOrForward = await this.getAndDecryptPreviousMessage((progress) => this.setQuoteLoaderProgress(progress + '%'));
+      try {
+        this.messageToReplyOrForward = await this.getAndDecryptPreviousMessage((progress) => this.setQuoteLoaderProgress(progress + '%'));
+      } catch (e) {
+        Catch.reportErr(e);
+        await Ui.modal.error(`The error occured while getting ${method === 'reply' ? 'replied' : 'forwarded'} message. The messasge won't be included in new message.`);
```

Comment:
Here at the end you can add `\n${Api.err.eli5(e)}`

eli5 = "Explain like I'm 5". It will add user-friendly explanation of what exactly went wrong.

Reply:
aha, I was thinking about why it's called `eli5` 
Now, I know :)
I've added it